### ISSUE TITLE
A11y_Chat window_HighContrast:Cross button is not visible in desert HC mode.

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#4559]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Users viewing the page in HC mode will face difficulties.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator, Windows: Chrome with NVDA, Windows: Chrome/Firefox with JAWS

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Contrast ratio, Forced colors (high contrast mode)

### What is the public URL for the website?

https://web.powerva.microsoft.com/environments/839eace6-59ab-4243-97ec-a5b8fcc104e4/bots/107725f1-d827-41a4-8375-c91d1a97affc

### How to reproduce the issue?

Pre-Requisite:
Set Contrast theme to desert theme.

1. Launch the application using URL: [Home - Test | Power Virtual Agents (microsoft.com)](https://web.powerva.microsoft.com/environments/839eace6-59ab-4243-97ec-a5b8fcc104e4/bots/107725f1-d827-41a4-8375-c91d1a97affc)
1. Tab till chat box and press enter to open it.
1. Press tab key to navigate to cross button.

### What do you expect?

Cross button should be clearly visible in HC mode.

### What actually happened?

Cross button is not visible in desert HC mode.

### Do you have any screenshots or recordings to repro the issue?

![image](https://user-images.githubusercontent.com/1622400/206585414-0cfdcdb3-b21a-4e81-841a-d6cb599800da.png)

### MAS reference

https://aka.ms/MAS4.3.1

### Additional context

The color use in SVG should be a CSS system color of "text color"
